trade-finance-manager-api/src/v1/controllers/amendment.controller.js

```diff
@@ -47,6 +49,19 @@ const sendAmendmentEmail = async (amendmentId, facilityId) => {
 const updateFacilityAmendment = async (req, res) => {
   const { amendmentId, facilityId } = req.params;
   const payload = req.body;
+  if (payload.createTasks && payload.submittedByPim) {
+    const tasks = createAmendmentTasks(payload.requireUkefApproval);
+    payload.tasks = tasks;
+    delete payload.createTasks;
+    delete payload.requireUkefApproval;
+  }
+
+  if (payload?.taskUpdate?.updateTask) {
+    const tasks = await updateAmendmentTask(facilityId, amendmentId, payload.taskUpdate);
```

Comment:
not a showstopper at all: can be changed later, could this be updateAmendmentTasks instead?

Reply:
good shout! - let's do it in a separate PR though